plugins/catalog-backend/package.json

```diff
@@ -38,6 +38,7 @@
     "@octokit/graphql": "^4.5.8",
     "@types/express": "^4.17.6",
     "@types/ldapjs": "^1.0.9",
+    "aws-sdk": "^2.840.0",
```

Comment:
We should remove the V3 dependency here.

`"@aws-sdk/client-organizations": "^3.2.0"`

Reply:
I realise there's still the Credential Provider Node used in the Kubernetes plugin